app/redux/ducks/translations.js

```diff
@@ -40,7 +40,7 @@ const initialState = {
   rtl: false,
   strings: {
     project: {},
-    workflow: {},
+    workflow: { id: null },
```

Comment:
Why was this `id: null` added?

Reply:
If there's no translation for the workflow I wanted to provide a default or null value so when on classification completion `translations.strings.workflow.id` is saved we'll know there wasn't a translation, as opposed to just not saving anything and wondering if it worked at all. So I added that default id of null in the duck as well as the classifier component.

Reply:
Does the API client record a value if you pass `null`? I thought it removed the key in that case. I could be wrong though.

Reply:
Looks like if no id is present, `workflow_translation_id: null` is recorded in the metadata.

Reply:
I just did a few classifications on a testing project without any workflow translations and ran a data export and I think we're ok:
<img width="1025" alt="screen shot 2018-12-07 at 11 00 45 am" src="https://user-images.githubusercontent.com/12118751/49661716-763d9180-fa0f-11e8-8577-8c258f743e9f.png">

Reply:
Cool. Sounds like we didn't roll out translations on staging though. All projects should have them at this point, even if it's just the one English translation that you get by default.